lib/TAP/Harness/Runtime.pm

```diff
@@ -5,7 +5,7 @@ use warnings;
 
 use base q (Exporter);
 
-use constant HAS_TIME_HIRES => !! eval { use Time::HiRes qw (time); 1 };
+use constant HAS_TIME_HIRES => !! eval q { use Time::HiRes qw (time); 1 };
```

Comment:
This changes the eval from a block style evaluation to a string, which is not a good change.

Reply:
Block eval would not be able to test a use statement, but this can easily be done with a block eval: `eval { require Time::HiRes; Time::HiRes->import('time'); 1 }`

Reply:
@karenetheridge original code had string evaluation.

I'm using form `q { }` for string containing code (hence `{` as paired delimiter).

commit introducing this change is `fixup!` of original commit. I'm using such approach to show reaction to code review comment, referring fixup commit(s) dealing with given issue raised in given comment.
(when I was taught this approach I hesitated but at the end, in many PR-review cycles, I found out that review by commit and reacting with  fixups saved lot of time especially on reviewer's side)

@Grinnz my aim was only to extract existing runtime detection into dedicated library, but your suggestion is too good to be omitted ... 033f677